sway-core/src/asm_generation/fuel/data_section.rs

```diff
@@ -14,7 +14,7 @@ pub struct Entry {
     padding: Padding,
     // It is assumed, for now, that only configuration-time constants have a name. Otherwise, this
     // is `None`.
-    name: Option<String>,
+    pub name: Option<String>,
```

Comment:
Can this be `pub(crate)`?

Reply:
This will not be needed anymore for this PR.